composer.json

```diff
@@ -38,7 +38,7 @@
         "minimaxir/big-list-of-naughty-strings": "dev-master",
         "pear/console_table": "~1.3",
         "phing/phing": "^2.15.0",
-        "php": ">=5.5.9",
+        "php": ">=7.0",
```

Comment:
OK moving back to 5.5.9 :(

I made some PHP 7 only changes, I'll go through the diff to update them.